### PR TITLE
Code changes requested in https://github.com/dotnet/docs/pull/9817

### PR DESCRIPTION
**Pull request by v-thepet**
## Summary

@rpetrusha requested the following code changes in https://github.com/dotnet/docs/pull/9817. (Requested content changes are made in that PR.) These are the requested code changes:

In how-to-read-characters-from-a-string.md and how-to-write-characters-to-a-string.md (same samples): 
(../../../samples/snippets/csharp/VS_Snippets_CLR/conceptual.stringreader/cs/source2.cs#2)
(../../../samples/snippets/visualstudio/VS_Snippets_CLR/conceptual.stringreader/vb/source2.cs#2)

The example won't compile, since it's a single code file. This needs to be a complete WPF project, which would have to go in a new directory. Then display mainwindow.cs and mainwindow.vb.
The list of imported namespaces in the VB example is also incomplete when compared with C#.

(../../../samples/snippets/csharp/VS_Snippets_CLR/conceptual.basicio.textfiles/cs/source6.cs#6)]
(../../../samples/snippets/visualbasic/VS_Snippets_CLR/conceptual.basicio.textfiles/vb/source6.cs#6)]

This also needs to be compilable. Create a separate directory for the WPF project.
The list of VB imported namespaces is also incomplete.

(../../../samples/snippets/visualbasic/VS_Snippets_CLR_System/system.IO.StreamReader/VB/source2.vb#20)

The VB code example includes "Imports System", which is unnecessary. It would be nice to get rid of them in a PR in dotnet/samples.

(../../../samples/snippets/csharp/VS_Snippets_CLR/conceptual.basicio.textfiles/cs/source2.cs#2)
(../../../samples/snippets/visualbasic/VS_Snippets_CLR/conceptual.basicio.textfiles/vb/source2.vb#2)
It would be good to include output in the examples.
the VB code also imports the System namespace.

(../../../samples/snippets/csharp/VS_Snippets_CLR/conceptual.basicio.textfiles/cs/source3.cs#3)]
(../../../samples/snippets/csharp/VS_Snippets_CLR/conceptual.basicio.textfiles/cs/source6.cs#6)]
Rather than handling Exception, the examples should handle IOException and FileNotFoundException.

(Also interpolated some strings because it now bothers me when they are not.)

Part of User Story https://mseng.visualstudio.com/TechnicalContent/_queries/edit/1374049/?triage=true.

### CODE REVIEW

**Comment by v-thepet** [comment]
@rpetrusha - I think this addresses the comments except for the ones about the WPF apps. 
I seem to have succeeded in getting (../../../samples/snippets/csharp/VS_Snippets_CLR/conceptual.basicio.textfiles/cs/source6.cs#6)]
(../../../samples/snippets/visualbasic/VS_Snippets_CLR/conceptual.basicio.textfiles/vb/source6.cs#6)] 
to compile, but am not sure what it does or should do. Also don't understand about creating a new directory etc. - is this sample supposed to do that? I am running these within a WPF app (MainWindow.xaml.cs), but not sure how that is supposed to affect the sample itself.

Have not made any headway with the other WPF sample at all, but will keep trying. 
(../../../samples/snippets/csharp/VS_Snippets_CLR/conceptual.stringreader/cs/source2.cs#2)
(../../../samples/snippets/visualstudio/VS_Snippets_CLR/conceptual.stringreader/vb/source2.cs#2)
"UserInput" and "Result" are not recognized.

File: snippets/visualbasic/VS_Snippets_CLR/conceptual.basicio.textfiles/vb/source3.vb [hunk 1]
```diff
@@ -12,7 +11,7 @@ Class Test
                 line = sr.ReadToEnd()
```
Comment (rpetrusha) [inline snippets/visualbasic/VS_Snippets_CLR/conceptual.basicio.textfiles/vb/source3.vb]: line 9 should be deleted, and this line should be `Dim line = sr.ReadToEnd()`

File: snippets/visualbasic/VS_Snippets_CLR/conceptual.basicio.textfiles/vb/source6.vb [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
 Imports System.IO
+Imports System.Windows
 
 Class MainWindow
     Private Async Sub ReadFileButton_Click(sender As Object, e As RoutedEventArgs)
         Try
             Using sr As StreamReader = New StreamReader("TestFile.txt")
                 Dim line = Await sr.ReadToEndAsync()
-                ResultBlock.Text = line
+                Console.WriteLine(line)
```
Comment (rpetrusha) [inline snippets/visualbasic/VS_Snippets_CLR/conceptual.basicio.textfiles/vb/source6.vb]: Console.WriteLine here and in line 12 is not supported.

**Comment by v-thepet** [comment]
Addressing one missed change request from #9817. 
(../../../samples/snippets/csharp/VS_Snippets_CLR_System/system.IO.BinaryReaderWriter/CS/source6.cs#7)
(../../../samples/snippets/visualbasic/VS_Snippets_CLR_System/system.IO.BinaryReaderWriter/VB/source6.vb#7)  
The VB example again imports System. This example would also benefit from displaying output.

**Comment by v-thepet** [comment]
@rpetrusha - Thanks for the help. I was always running these in WPF projects, but didn't realize I needed to change the XAML files too. It will probably be helpful to include the MainWindow.xaml files in the topics, so once this is merged I can change the topics to do that (will need to update the links regardless). 
The new apps are at samples\snippets\csharp [and visualbasic]\VS_Snippets_Wpf\StringReaderWriter and samples\snippets\csharp [and visualbasic]\VS_Snippets_Wpf\TextFiles. I added the xaml and cs files and the csproj file since that's what most of the other projects in the folder were doing. 
Please review. Thanks.

**Pull request merged**